### PR TITLE
Fix failing regress

### PR DESCRIPTION
**Pull request by JumpingYang001**
New PR #48

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -99,6 +99,7 @@ build_omi
 copy_version_file
 
 echo "Building OMI Python Script provider"
+export CPPFLAGS="-fpermissive"
```
Comment (c64cosmin) [inline build.sh:102]: This should be removed!

**Review by c64cosmin — COMMENTED** [review]
[This](cbcb9974b37bef1448fd453dbf13d6644bf6209a) commit is the one which fixes the Travis state.

**Comment by JumpingYang001** [comment]
Add it in #47 and close this one.

File: build.sh (hunk 1 above)
Comment (JumpingYang001) [reply build.sh:102]: ho, just see this now, but this is in your commit...
you can remove it next time, thanks.

**Pull request closed without merging**